produce/lib/produce/options.rb

```diff
@@ -137,7 +137,19 @@ def self.available_options
                                      default_value: CredentialsManager::AppfileConfig.try_fetch_value(:itc_team_name),
                                      verify_block: proc do |value|
                                        ENV["FASTLANE_ITC_TEAM_NAME"] = value.to_s
-                                     end)
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :merchant_name,
+                                      short_option: "-r",
+                                      env_name: "PRODUCE_MERCHANT_NAME",
+                                      description: "Name for the merchant that is created",
+                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :merchant_identifier,
```

Comment:
I don't have the full context here, but from my understanding this would also show up when running `fastlane produce --help`, right? If so, what happens if the user provides this for the regular produce command, and not the one for merchants? You mention the other `--help` command in the README, doesn't this show the same options than the main one? Just asking some questions here, I haven't checked myself.